components/x11/xlock/Makefile

```diff
@@ -11,51 +11,36 @@
 
 #
 # Copyright 2015 Aurelien Larcher
+# Copyright 2019 Michal Nowak
 #
 
 include ../../../make-rules/shared-macros.mk
 
-COMPONENT_NAME=           xlock
-COMPONENT_VERSION=        7.5
-COMPONENT_REVISION=       1
-COMPONENT_FMRI=           x11/xlock
+COMPONENT_NAME=		xlock
+COMPONENT_VERSION=	7.5
+COMPONENT_REVISION=	2
+COMPONENT_FMRI=		x11/xlock
 COMPONENT_CLASSIFICATION= System/X11
-COMPONENT_SUMMARY=        xlock - screen lock program
-COMPONENT_SRC=            src
-COMPONENT_PROJECT_URL=    https://hg.java.net/hg/solaris-x11~x-s12-clone
-COMPONENT_LICENSE=        MIT License
+COMPONENT_SUMMARY=	xlock - screen lock program
+COMPONENT_SRC=		src
+COMPONENT_PROJECT_URL=	https://github.com/oracle/solaris-userland/tree/master/components/x11/app/xlock/sun-src
+COMPONENT_LICENSE=	MIT License
 
 include $(WS_MAKE_RULES)/prep.mk
 include $(WS_MAKE_RULES)/justmake.mk
 include $(WS_MAKE_RULES)/ips.mk
 
-PATH=/usr/gnu/bin:/usr/bin
+COMPONENT_BUILD_ENV += $(COMPONENT_COMMON_ENV)
+COMPONENT_INSTALL_ENV += INSTALL="$(INSTALL) -D"
+COMPONENT_INSTALL_ENV += PREFIX=$(USRDIR)
 
-COMPONENT_COMMON_ENV = CC=$(CC)
-COMPONENT_COMMON_ENV+= CFLAGS="$(CFLAGS)"
-COMPONENT_COMMON_ENV+= LDFLAGS="$(LDFLAGS)"
+build: $(BUILD_64)
 
-XLOCK_BINDIR.32= $(USRBINDIR)
-XLOCK_BINDIR.64= $(USRBINDIR64)
-
-COMPONENT_BUILD_ENV+= $(COMPONENT_COMMON_ENV)
-COMPONENT_BUILD_ARGS= VPATH=$(SOURCE_DIR) -f $(SOURCE_DIR)/Makefile
-COMPONENT_INSTALL_ENV+= INSTALL=$(INSTALL)
-COMPONENT_INSTALL_ENV+= DESTDIR=$(PROTO_DIR)
-COMPONENT_INSTALL_ENV+= PREFIX=$(USRDIR)
-COMPONENT_INSTALL_ENV+= bindir=$(XLOCK_BINDIR.$(BITS))
-COMPONENT_INSTALL_ENV+= datadir=$(USRSHAREDIR)
-COMPONENT_INSTALL_ENV+= mandir=$(USRSHAREMANDIR)
-
-$(SOURCE_DIR)/.prep: $(MAKEFILE_PREREQ)
```

Comment:
Is this not needed anymore?

Reply:
`COMPONENT_COMMON_ENV` instances were actually needed, I added them back as well as the `.prep` target.